src/tribler-gui/tribler_gui/event_request_manager.py

```diff
@@ -69,20 +75,23 @@ def events_start_received(self, event_dict):
                 SentryReporter.set_user(public_key.encode('utf-8'))
 
     def on_error(self, error, reschedule_on_err):
-        self._logger.info(f"Got Tribler core error: {error}")
+        if error == QNetworkReply.ConnectionRefusedError:
+            if self.remaining_connection_attempts % 10 == 0:
+                self._logger.info("Tribler Core refused connection, retrying...")
```

Comment:
**Issue** it is unclear, why should we log each 10th attempt?
From my POV  it is better either to remove logging (line 80) or to replace it by:

```suggestion
            self._logger.debug("Tribler Core refused connection, retrying...")
```

Reply:
If I understood you correctly before, your original concern was about the logger spamming the breadcrumbs list. Well, logging each 10th attempt solves that problem. Seeing the connection attempts at the "Info" level is really useful during development.

 I'm no expert with Sentry internals, but if you think changing the logging level to Debug mode will solve the breadcrumbs spam problem, so be it.

Reply:
Breadcrumbs are configurable. Now we use these settings:
https://github.com/Tribler/tribler/blob/151e7ab936187717786b05736cd0dcf4b455340a/src/tribler-common/tribler_common/sentry_reporter/sentry_reporter.py#L114-L117

Reply:
BTW, wouldn't enabling Sentry to capture Debug-level logs enhance our ability to debug problems? Or that will produce too much output?

Reply:
I'm not sure, but I guess so :)